ansible_collections/arista/avd/roles/eos_designs/schemas/schema_fragments/defs_node_type.schema.yml

```diff
@@ -28,15 +28,14 @@ $defs:
               System MAC Address in this following format: "xx:xx:xx:xx:xx:xx".
               Set to the same MAC address as available in "show version" on the device.
               "system_mac_address" can also be set directly as a hostvar.
-              If both are set, the setting under "Fabric Topology" takes precedence.
+              If both are set, the setting under node type settings takes precedence.
           serial_number:
             type: str
             description: |
-              Set to the Serial Number of the device
-              For  now only used for documentation purpose in the fabric documentation
-              and part of the structured_config
+              Set to the Serial Number of the device.
+              Only used for documentation purpose in the fabric documentation and part of the structured_config.
               "serial_number" can also be set directly as a hostvar.
-              If both are set, the setting under "Fabric Topology" takes precedence.
+              If both are set, the setting under node type settings takes precedence.
```

Comment:
Doesn't hostvar take precedence?

Reply:
The unfortunately varies from var to var.